client/src/components/StepWizard/WizardCheckboxItem.js

```diff
@@ -42,6 +42,11 @@ export const WizardCheckboxItem = styled(CheckboxItem)`
       width: 0.7rem;
     }
   }
+   > .text {
+   color: #000;
```

Comment:
Use theme color.

Reply:
The `Checkbox.CheckboxItem` component was setting the text color to black. If I use one of the theme colors, the text will be either blue or grey:
![](https://i.imgur.com/cSjUuDn.png) lightGray text
![](https://i.imgur.com/cFPvrM1.png) royalBlue text

@silenceway  the text for all the other similar text boxes is black (see pages in the `/need-help` route ). 
@ppagarawal  Is using the blue theme color here appropriate or should it remain black?  I just joined this project so please advise. Thanks! All